### ISSUE TITLE
how to remove the @Generated annotation

### ISSUE THREAD

**Issue by TacticalTwerking** [joelittlejohn/jsonschema2pojo#834]
At first ,thank you for your amazing library, it's powerful and saving people.

right now i am using this library in gradle with dependency  **jsonschema2pojo-gradle-plugin version: '1.0.0-alpha1'**

**@Generated("org.jsonschema2pojo")**

i'm confused because this(above line) appear in every class that generated ,and i don't want more dependencies. So after some google search .there's an issue post kind like some solutions  contained [issue577](https://github.com/joelittlejohn/jsonschema2pojo/issues/577)

and i tried to change the dependency version from 1.0.0-alpha to both of 0.4.26 and 0.4.24
but it doesn't work for me at all 

edit : almost forgot to mention this :
how can i make them extends from some class programmatically (maybe i can specify a class name)?

any help thanks

**Comment by joelittlejohn** [comment]
Since 0.4.26 this project has not added that annotation. Make sure you have fully cleaned your project, and make sure you are using the correct version, and you will not see these annotations.

**Comment by TacticalTwerking** [comment]
Sorry mate, not sure if i was using the incorrect version .this is my gradle dependency:

dependencies {
compile group: 'org.jsonschema2pojo', name: 'jsonschema2pojo-gradle-plugin', version: '1.0.0-alpha1'
}

**Comment by joelittlejohn** [comment]
0.4.26 was released 1 Sep 2016, this is when we stopped adding the `@Generated` annotation. 1.0.0-alpha1 was released just 8 days ago so yes, 1.0.0-alpha1 does not add `@Generated`.

I think somehow you either have sources that were generated by a different version of jsonschema2pojo still hanging around from a different invocation, or your gradle build includes some other config somewhere that is causing a different version to be used. 

If you search the codebase of this project for 'javax.annotation' you'll see the only annotations we use are Nullable and Nonnull. There is no reference to `java.annotation.Generated` in this project's source code.
